modules/statusline/lualine/lualine.nix

```diff
@@ -276,47 +286,111 @@ in {
                 dos = '',  -- e70f
                 mac = '',  -- e711
               },
-            },
-          }
-        '';
+            }
+          ''
+        ];
+      };
+    };
+    extraActiveSection = {
+      a = mkOption {
+        type = types.listOf types.str;
```

Comment:
A minor nitpick, but I'd rather use `with types; listOf str` in option definitions.